### PR TITLE
Fix putting elements under <p>

### PR DESCRIPTION
**Pull request by TomNUSDS**
- Fixed the errors around putting other html elements inside `<p>` 
- Added Jest test to catch this mistake in the future. It does it by watching console.error().To add in fixing the bug the test's emulated `console.error()` are echoed to the jest console.error() so they are not lost. I tested this by putting a `<h1>` back into the `<p>` and it caught it.
 
We should figure out how to use this technique in more places?

### CODE REVIEW

**Review by vim-usds — COMMENTED** [review]
Any luck on getting the console.error abstracted to jest.config?

File: client/src/components/scoreStepsList.test.tsx [hunk 1]
```diff
@@ -3,6 +3,12 @@ import {render} from '@testing-library/react';
 import ScoreStepsList from './scoreStepsList';
 import {LocalizedComponent} from '../test/testHelpers';
 
+beforeAll(() => {
+  jest.spyOn(global.console, 'error').mockImplementation((...params) => {
```
[the remark this answers is not in the document]
Comment (TomNUSDS) [reply client/src/components/scoreStepsList.test.tsx:7]: I was thinking of putting it as "page level" tests (which will catch any errors in sub-components.

But that can be a separate task?  I'd really like to get this change merged.

[the remark this answers is not in the document]
Comment (vim-usds) [reply client/src/components/scoreStepsList.test.tsx:7]: I hear ya, can you create the ticket and add it as a Todo in the code?

// Todo : look into abstracting console.error to all test files, see ticket here: ......

This ticket could be a great `first issue` and `help wanted`. We can then merge it.

**Comment by TomNUSDS** [comment]
Close/reopen to generate deploy link for design.

**Comment by TomNUSDS** [comment]
Close/reopen to generate deploy link for design.

**Comment by TomNUSDS** [comment]
@janelien Design sanity check? https://d2zjid6n5ja2pt.cloudfront.net/justice40-tool/aa2826/en

File: client/src/components/scoreStepsList.test.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (TomNUSDS) [reply client/src/components/scoreStepsList.test.tsx:7]: Created a task: https://github.com/usds/justice40-tool/issues/550

**Comment by vim-usds** [comment]
> @janelien Design sanity check? https://d2zjid6n5ja2pt.cloudfront.net/justice40-tool/aa2826/en

Hey Tom! Since the PR didn't update with the PR'd URL, did you get the hash from the deploy logs? This link totally looks right, it's just that when I look at the deploy logs, the hash I see is different? You might have found a way around it! Just curious.

**Comment by vim-usds** [comment]
Also I think the design asks for Alert to only show one Alert and the cards have card headings.. I thought you put that in? I'm wondering about the hash of the above link. Hmm

**Comment by vim-usds** [comment]
> @janelien Design sanity check? https://d2zjid6n5ja2pt.cloudfront.net/justice40-tool/aa2826/en

This confused me for a second because I thought this had to do with this ticket. But it's actually in reference to the ABout page PR. Got it now!

**Comment by janelien** [comment]
Some thoughts:
- In the mobile mockup, I had the graphics above the text so it wouldn't get too squished. Is it possible to have it lay out that way in mobile?
- For the "Check it out on Github" link, could we have it open in a new tab? 
- Not sure if the alerts are part of this QA, but the "limited data sources" alert shouldn't be on the About page. 

Otherwise, looks good to me! I know we're continuing to discuss typography and spacing, so there may be some future tickets around those things. Thanks for your flexibility on this page! It's gone through a number of iterations, so appreciate the continuous updates!

File: client/src/components/scoreStepsList.test.tsx [hunk 2]
```diff
@@ -3,6 +3,12 @@ import {render} from '@testing-library/react';
 import ScoreStepsList from './scoreStepsList';
 import {LocalizedComponent} from '../test/testHelpers';
 
+beforeAll(() => {
```
Comment (vim-usds) [inline client/src/components/scoreStepsList.test.tsx:6]: Can we add the following to right above line 6?

// TODO: Move this to a location that will detect on all tests
// See ticket: https://github.com/usds/justice40-tool/issues/550

**Review by vim-usds — COMMENTED** [review]
Just a quick addition of a comment in the code.

File: client/src/components/scoreStepsList.test.tsx (hunk 2 above)
Comment (TomNUSDS) [reply client/src/components/scoreStepsList.test.tsx:6]: You should try out the "code suggestion" github feature! 
image

**Comment by TomNUSDS** [comment]
@janelien 
> * In the mobile mockup, I had the graphics above the text so it wouldn't get too squished. Is it possible to have it lay out that way in mobile?

I looked into it, but it seemed to fit ok even on the smallest mobile? 
image

And there's some talk of turning this whole logo into an svg image with text, so I punted the extra work. 

Is it ok to create another issue to do this work in a different PR?

> * For the "Check it out on Github" link, could we have it open in a new tab?

Will do. Technically, I think we should have all "off site" links go into a new tab.

> * Not sure if the alerts are part of this QA, but the "limited data sources" alert shouldn't be on the About page.

OK. Will fix this as well.

**Review by vim-usds — APPROVED** [review]
LGTM!

**Pull request merged**